RUFAS/routines/field/crop/crop_management.py

```diff
@@ -299,6 +299,21 @@ def _record_yield(self, field_name: str, field_size: float, year: int, day: int)
             Julian day on which this harvest occurred.
 
         """
+        units = {
+            "crop": "unitless",
+            "wet_yield": "kg/ha",
+            "dry_yield": "kg/ha",
+            "nitrogen": "kg/ha",
+            "phosphorus": "kg/ha",
+            "yield_residue": "kg/ha",
+            "harvest_index": "unitless",
+            "planting_date": {
+                "year": "unitless",
+                "day": "unitless",
+            },
+            "harvest_date": {"year": "unitless", "day": "unitless"},
```

Comment:
```suggestion
            "wet_yield": " wet kg/ha",
            "dry_yield": " dry kg/ha",
            "nitrogen": "kg/ha",
            "phosphorus": "kg/ha",
            "yield_residue": "dry kg/ha",
            "harvest_index": "unitless",
            "planting_date": {
                "year": "year",
                "day": "day",
            },
            "harvest_date": {"year": "year", "day": "day"},
```
These units definitely feel redundant, but I think erring on the side of too explicit is better.